charts/latest/csi-driver-nfs/values.yaml

```diff
@@ -38,6 +38,7 @@ kubeletDir: /var/lib/kubelet
 controller:
   name: csi-nfs-controller
   replicas: 1
+  strategyType: RollingUpdate
```

Comment:
Thanks, I think we could change the default value as `Recreate`, that could avoid driver install issue when reinstall driver on a cluster with only a few nodes.

Reply:
Alright, I'm going to change it